### DISCUSSION TITLE
More Distributions Installations

### DISCUSSION THREAD

**Discussion by fbrissi** [kool-dev/kool#334] [Ideas]
I think is kool idea have more distribution installation. It's not complicated.

See:
dpkg: https://blog.knoldus.com/create-a-debian-package-using-dpkg-deb-tool/
rpm: https://opensource.com/article/18/9/how-build-rpm-packages
pkg: https://medium.com/swlh/the-easiest-way-to-build-macos-installer-for-your-application-34a11dd08744

**Comment by danielsuguimoto** [comment]
homebrew: https://docs.brew.sh/Formula-Cookbook

**Comment by fabriciojs** [comment]
For Windows, we have to:

- Think about Store - https://docs.microsoft.com/en-us/windows/uwp/publish/

- Sign the installer to have no security alerts - https://github.com/kool-dev/kool/issues/42#issuecomment-687940790

**Comment by fabriciojs** [comment]
For MacOS signed/notary - https://github.com/mitchellh/gon

https://developer.apple.com/developer-id/

**Comment by fabriciojs** [comment]
Moving this to a discussion, if there are interest/requests for more distribution means we then can evaluate and follow up.
